components/postgresql/postgresql.app.mjs

```diff
@@ -93,9 +93,12 @@ export default {
         ...(cert && {
           cert,
         }),
-        rejectUnauthorized: mode !== "skip_verification",
       };
 
+      if (mode === "verify_identity") {
+        ssl.rejectUnauthorized = true;
+      }
```

Comment:
I think this wouldn't allow setting `rejectUnauthorized` to `false`, which someone may want to do to connect without CA verification to a server that requires SSL. (It looks like the `pg` package [sets `rejectUnauthorized` to `true` by default](https://github.com/brianc/node-postgres/blob/373093d176f0ceb0266333680b20d0acf38004e3/docs/pages/announcements.mdx#pg80-release) for SSL connections.)

Maybe set `ssl.rejectUnauthorized` if `mode` is not null, and omit it otherwise? For example:
```js
      if (mode) {
        ssl.rejectUnauthorized = mode === "verify_identity";
      }
```